airflow/www/static/css/bootstrap-theme.css

```diff
@@ -1576,6 +1576,7 @@ th {
   line-height: 1.428571429;
   vertical-align: middle;
   border-top: 1px solid #ddd;
+  white-space: nowrap;
```

Comment:
I think you'd probably want to move this down to the next block so that it is only applied to `th` elements, not the `td`s.